data_steward/test/unit_test/combine_ehr_rdr_test.py

```diff
@@ -124,23 +126,84 @@ def test_copy_rdr_tables(self):
             self.assertEqual(rdr_count, combined_count,
                              msg_fmt.format(table=table, rdr_count=rdr_count, combined_count=combined_count))
 
+    def test_ehr_person_to_observation(self):
+        # ehr person table converts to observation records
+        self.load_dataset_from_files(self.combined_dataset_id, test_util.NYC_FIVE_PERSONS_PATH)
```

Comment:
@aahladmanas is it sufficient for this test to start with empty tables instead? Loading from files takes a long time. I will make the change if you think that'd be ok.